simtools/applications/plot_simtel_histograms.py

```diff
@@ -68,7 +68,7 @@ def main():
     logger = logging.getLogger()
     logger.setLevel(gen.get_log_level_from_user(args_dict["log_level"]))
     n_lists = len(args_dict["hist_file_names"])
```

Comment:
Suggest to condense this into one sentence: `Opening {n_lists} histogram files` (don’t call it hist_file_names, this is hard to understand)

Reply:
Good point. I updated it to `Opening {n_lists} files` as the files can also be text files with lists of histogram files.